addon/templates/components/frost-pick-list.hbs

```diff
@@ -1,2 +1,54 @@
-ember-frost-pick-list component
-{{yield}}
\ No newline at end of file
+<div class="a-row">
+  {{frost-text
+    hook='filter-text'
```

Comment:
can we concat the hook coming in (set by the user) here? so like `hook=(concat hook '-filterText')`
Camelise the local hook name.